src/pages/ConfirmPage.js

```diff
@@ -1,18 +1,76 @@
 import React, { Component } from 'react';
-import { Grid, Row, Col, Alert } from 'react-bootstrap';
-import { Navbar } from '../components'
-
-const ConfirmPage = () => {
-  return (
-    <Grid fluid>
-      <Navbar />
-      <Row>
-        <Col md={8} mdOffset={2}>
-          <Alert bsStyle="success">Votre commande est validée !</Alert>
-        </Col>
-      </Row>
-    </Grid>
-  )
+import { Grid, Row, Col, Alert, Button} from 'react-bootstrap';
+import { Navbar, Cart } from '../components';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { withRouter, Redirect } from 'react-router-dom';
+import moment from 'moment';
+import { resetCheckout } from "../actions/index";
+
+
+moment.locale('fr');
+
+
+class ConfirmPage extends Component {
+
+  onClick() {
+    this.props.actions.resetCheckout();
+  }
+
+  render () {
+
+    if (!this.props.order) {
+      return (<Redirect to={{ pathname: '/' }} />);
+    }
+
+    const deliveryTime = moment(this.props.deliveryDate).format('HH[h]mm');
+    const deliveryDate = moment(this.props.deliveryDate).format('dddd DD MMMM');
+    const deliveryIsToday = moment(this.props.deliveryDate).isSame(Date.now());
+    let deliveryDateText = !deliveryIsToday ? ' le ' + deliveryDate : '';
+
+    const orderUrl = this.props.client.httpBaseURL + this.props.order.publicUrl;
```

Comment:
In a more general comment about the syntax... (but I don't want to annoye you too much with that)
it is cool to unpack all your items props once at the beginning render function.

like here it would be
```
render () {
   const { client, deliveryDate, order: { publicUrl } } = this.props
   /* do some computation stuff given these keys */
   return (<div>
     /* define your render given these keys */
   </div>
   )
}

```

Reply:
Done